main.py

```diff
@@ -1,22 +1,34 @@
 from __future__ import unicode_literals
-import subprocess
-import hostnameSearch
-import usernameSearch
-import os
 from collections import defaultdict
-import config_properties
-import urllib
-import requests
+from distutils.version import StrictVersion
```

Comment:
Is this included in python 2.7, or does it have to be included in the workflow package download.